features/rerun_formatter.feature

```diff
@@ -85,3 +85,19 @@ Feature: Rerun Formatter
       | A - ambiguous |
       | B - pending   |
       | C - undefined |
+
+  Scenario: rerun file with trailing new line
+    Given a file named "@rerun.txt" with:
+      """
+      features/c.feature:2
+
+      """
+    When I run cucumber.js with `-f json @rerun.txt`
+    Then the exit status should be 0
+
+  Scenario: empty rerun file
```

Comment:
This is a change in behavior. Previously an empty rerun file also ran all the feature files in the current directory